CHANGELOG.md

```diff
@@ -7,12 +7,12 @@
 
 
 
-
```

Comment:
?

Reply:
What was this line with no content other than - ?